.travis.yml

```diff
@@ -48,7 +49,8 @@ install:
 script:
     - mkdir for_test
     - cd for_test
-    - pytest -v --cov=sktime_dl --pyargs ../sktime_dl
+    #- pytest -v --cov=sktime_dl --pyargs ../sktime_dl # 45mins Travis run
+    - pytest -v -m="not slow" --cov=sktime_dl --pyargs ../sktime_dl 
```

Comment:
Why do we need to flaky? What does the `-m="not slow"` do? Perhaps add a short comment here.

Reply:
Added comment to explain "slow", thanks.